### DISCUSSION TITLE
Welcome to Aeolus-AeroSketchPad Discussions!

### DISCUSSION THREAD

**Discussion by UweSchuster** [UweSchuster/Aeolus-AeroSketchPad#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by j2l** [comment]
Hello @UweSchuster and Aeolus team!
I discovered Aeolus through a youtube recommendation as I was crawling for CFD for my hobby project (an out-of-norms kite wing) and I was impressed by the quality of your app (and I tested a lot of them)!
I really appreciate your global approach to designing realistic wings from concept to detailed design, it really helps me define and validate my choices about the wing shape design.
I found a little bug I'll add as an issue here but I wish I could help you more. I'm not an engineer, more a marketing guy :smile:

**Reply by UweSchuster** [reply]
Thank you @j2l ! Also many thanks for reporting the popup issue. I will take care of it and fix it with the next release. :)
Please dont hesitate to let me know if you need any other feature or improvements.
Cheers! Uwe

**Reply by j2l** [reply]
Actually, I think of two features: 
- One simple and essential: being able to search and select an airfoil in geometry from the list (not just a dropdown that I can't enter text in), because names sometimes differ from airfoiltools and it's simply long to crawl the list and pinpoint the right one.
- One more advanced: providing Cl/Cd for each (if you have it, or the most interesting?) of your airfoils and Re, and being able to sort them, like on airfoiltools: 
![image](https://user-images.githubusercontent.com/65325/160594134-c2b69b30-b9e0-43a6-8a40-748a01349efc.png)
Heliciel/Mecaflux also has a searchable database that stores Xfoil (PACC) results for numerous profiles (I told you I'm using a lot of softwares).

**Reply by UweSchuster** [reply]
Thanks @j2l. Thats translated into issue #3. I agree that its tedious to scroll through that list and find the right airfoil. Room for improvement! Thanks again for your input!
